plugins/inputs/unbound/unbound.go

```diff
@@ -207,6 +216,7 @@ func init() {
 			UseSudo:     false,
 			Server:      "",
 			ThreadAsTag: false,
+			ConfigFile:  defaultConfigFile,
```

Comment:
Let's default this to "", so that `unbound` picks the default file.

Reply:
Thanks for your feedback. I've changed that.